src/typechecker.erl

```diff
@@ -3167,14 +3175,27 @@ check_clauses(Env, [], {var, _, TyVar}, Clauses) ->
     {Ty, VarBinds, Cs} = infer_clauses(Env, Clauses),
     {VarBinds, constraints:combine(constraints:upper(TyVar, Ty), Cs)};
 check_clauses(Env, ArgsTy, ResTy, Clauses) ->
-    {VarBindsList, Css, _RefinedArgsTy} =
+    {VarBindsList, Css, RefinedArgsTy} =
         lists:foldl(fun (Clause, {VBs, Css, RefinedArgsTy}) ->
                             {NewRefinedArgsTy, VB, Cs} =
                                 check_clause(Env, RefinedArgsTy, ResTy, Clause),
                             {[VB | VBs], [Cs | Css], NewRefinedArgsTy}
                     end,
                     {[], [], ArgsTy},
                     Clauses),
+    % Checking for exhaustive patternmatching
+    case {Env#env.exhaust
+	 ,ArgsTy =/= any andalso
```

Comment:
Indentation looks odd. Is it tabs?

Reply:
See below. Not sure what's going on...

Reply:
This looks fine if I go to the "Files changed" tab so it's probably just a bug in github.